app/views/templates/Layout.scala.html

```diff
@@ -33,19 +33,6 @@
 @(pageTitle: String, showBackLink: Boolean = true, timeout: Boolean = true, showSignOut: Boolean = true, showHmrcBanner: Boolean = false)(contentBlock: Html)(implicit rh: RequestHeader, messages: Messages)
 
 @head = {
```

Comment:
Given that `@head` is now empty HTML, you can remove this whole section further down:

```
    templateOverrides = TemplateOverrides(
        additionalHeadBlock = Some(head)
    ),
```

Reply:
Thank you. I've removed the redundant code in the latest commit.